spec/controllers/trips_controller_spec.rb

```diff
@@ -31,19 +31,24 @@
   # adjust the attributes here as well.
   let(:valid_attributes) {
     {destination: 'NYC Conference',
-     reason: 'Hana Things',
-     annotation: 'HANA pls',
-     signature: true,
-     user: @user}
+    reason: 'Hana Things',
+    annotation: 'HANA pls',
+    date_start: Date.today,
+    date_end: Date.today+2,
```

Comment:
Ich hab letztens mal mit jemandem vom SWA-Lehrstuhl geredet und das Ergebnis war, dass es wohl besser ist in Tests statische Daten zu benutzen und lieber Corner Cases (Leap Year, Jahreswechsel ...) abzufangen. Wenn ihr damit nichts rechnet, dann sollte das aber relativ egal sein, glaube ich.